smartmin/models.py

```diff
@@ -240,12 +247,12 @@ def import_xls(cls, filename, user, import_params, log=None, import_results=None
                         num_errors += 1
 
                 except SmartImportRowError as e:
-                    error_messages.append(dict(line=line_number+1, error=text_type(e)))
+                    error_messages.append(dict(line=line_number+1, error=six.text_type(e)))
```

Comment:
Just making this consistent with how the method is called elsewhere in this file